### PR TITLE
Handholes alternative solution, less issues version™

### PR DESCRIPTION
**Pull request by ElorgRHG**
## What Does This PR Do

Alternative to: https://github.com/ParadiseSS13/Paradise/pull/19614
Fixes: https://github.com/ParadiseSS13/Paradise/issues/19825

So anyways, got bored of a new issue being found with the handholes PR every second week, decided on a new solution entirely.

Instead of taking the hand of the mob, copying an overlaying it on top of jumpsuit, this solution does just that... but with masks, hopefully a more futureproofed, less issue-prone solution. Example of futureproofing, this PR will work without any large changes needed IF a potential hand and arm merge happens, food for thought.

First thing it fixes, golem hands, and since its an alternative PR to the one that fixed hand markings, it also fixes hand markings.

## Why It's Good For The Game

Better solution?

## Images of changes

There's not much to show here but

image

image

image

image

## Testing

Arranged `Images of changes`, spawned as a golem, had proper golem hands.

## Changelog
:cl:
fix: Actually fixed golem hands.
fix: Husks will no longer have non-husked hands on their sprite.
/:cl:

### CODE REVIEW

**Review by SteelSlayer — APPROVED** [review]

**Comment by Bm0n** [comment]
The hands on the sec record pictures are messed up I think from this PR
image

**Review by hal9000PR — CHANGES_REQUESTED** [review]
you are gonna need to either, make `get_id_photo` work with `getflaticon` OR make it introduce the handhole masks. the second would likely be easier.

**Comment by ElorgRHG** [comment]
> The hands on the sec record pictures are messed up I think from this PR

Fixed
image

File: code/datums/datacore.dm [hunk 1]
```diff
@@ -349,6 +349,8 @@ GLOBAL_VAR_INIT(record_id_num, 1001)
 
 	preview_icon.Blend(face_s, ICON_OVERLAY)
 
+	var/icon/hands_icon = icon(preview_icon)
+	hands_icon.Blend(icon('icons/mob/clothing/masking_helpers.dmi', "l_hand_mask"), ICON_MULTIPLY)
```
Comment (hal9000PR) [inline code/datums/datacore.dm:353]: will this work properly with different species? like vox?

Comment (ElorgRHG) [reply code/datums/datacore.dm:353]: Its the same case as with character preview in the lobby, even before the PR, so it won't

image

image


making it work with species variants would be an infinitely larger refactor, that, and I hope you won't blame me for that, I don't have time, energy, nor perhaps the skill to undertake, sorry.

**Review by hal9000PR — APPROVED** [review]
we ball

**Pull request merged**